src/v3/src/types/stepTransformer.ts

```diff
@@ -20,7 +20,7 @@ type IdxStepTransformerOptions = {
   transaction: IdxTransaction;
   prevTransaction?: IdxTransaction;
   formBag: FormBag;
-  widgetProps: WidgetProps;
+  widgetProps?: WidgetProps;
```

Comment:
Is this intentional? I believe for all transformers we will _want_ to ensure widgetProps are provided downstream. This could cause issues later if a transformer expects the props but they are not present.

Reply:
Maybe this was done to remove the warning from tests? If so, you can just provide a mock empty object, but we definitely would not want to make this property optional.